### DISCUSSION TITLE
UPDATE: VENUE IS FULL Invitation: Mar 25, 2024 DataFusion meetup before the DataCouncil March 2024 in Austin TX

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#8522] [General]
# VENUS IS FULL; The venue can hold up to 30 people. We now have a waiting list for additional attendees

This is the follow on to https://github.com/apache/arrow-datafusion/discussions/8152

# Invititation 🎉 
I would like to officially invite you to join the first ever (that I know of) in person DataFusion meetup
* **Time**: Monday March 25th 2024 10 am to 5 pm 
* **Location**:  [WeWork 316 W 12th St. Austin TX, USA](https://www.google.com/maps/place/316+W+12th+St,+Austin,+TX+78701/@30.2749951,-97.7483294,17z) (10 minute walk from the  [DataCouncil '24](https://www.datacouncil.ai/) conference venue, [AT&T Hotel & Conference Center in Austin TX, USA](https://www.google.com/maps/place/AT%26T+Hotel+and+Conference+Center/@30.2816295,-97.7404408,15z/data=!4m2!3m1!1s0x0:0x7ef52b1ad3321879?sa=X&ved=2ahUKEwin17CNyK78AhWjPH0KHVC3Cq4Q_BJ6BAh0EAc)

The objective of this meetup is to continue building the community, discuss common goals, and challenges and brainstorm ways to grow and nurture the community.

# Agenda
(Updated):  [Agenda Document](https://docs.google.com/document/d/1wbD0NmChEMPy_CQSU7NHZjU4JmHXH7xYUGRJ6NPG1mc)

Please let me know by email to alamb@influxdata.com as soon as possible and by March 18th, 2024 at the latest if you would be able to join us.

# Note
It is important to note that we will not "decide" anything at the meeting, and will follow the same process as we do for other consensus lead decisions, including publicly posting all materials, and providing a video link/recordings for anyone who would like to attend but can not travel.

Also I am keeping this invitation on github (rather than, say meetup.com) to keep the attendance focused on users and contributors rather than a broader community of interest

**Comment by alamb** [comment]
Here is a proposed agenda (details in the [Agenda Document](https://docs.google.com/document/d/1wbD0NmChEMPy_CQSU7NHZjU4JmHXH7xYUGRJ6NPG1mc))

12:00 - 1:00  Lunch/Networking
1:00 - 1:30: Welcome / Introductions:  @alamb 
1:30 - 3:00: 6 10-15 minute talks about “How do you DataFusion”?
3:00 - 3:15: Coffee / Email Break
3:15 - 4:30: Breakouts / discussions
4:30 - 5:00: Close / Conclusion: @alamb 

As a reminder, all content should be publicly shareable and we will endeavor to provide zoom links / recordings of all talks as much as possible

**Comment by alamb** [comment]
**Call for Presenters**: "How do you DataFusion" will hopefully be several 10-15 minute low key introductions to your project/product, how you use DataFusion, and any thoughts you may have about what , found challenging, and any things you are working on / thinking about. Maybe a few slides.

**Comment by alamb** [comment]
BTW, If you plan to attend, please RSVP using the details above. If you don't I can not guarantee we will have space for you.

**Comment by alamb** [comment]
NOTE we are almost at VENUE capacity; We may not have space for additional attendees.

**Comment by stormasm** [comment]
Are you planning on recording the event and putting it up on youtube afterwards so folks can watch it later ?

**Comment by alamb** [comment]
I plan to do so as much as I can

**Comment by alamb** [comment]
The venue is full -- I'll keep a waiting list for additional attendees

**Comment by alamb** [comment]
I will ensure that all slides used at the meetup are shared publically afterwards. I will attempt to ensure all the talks are recorded as well as providing a ZOOM link if possible. However, the availability of the live zoom will depend on the venue setup (I'll know the day before)

**Comment by alamb** [comment]
For anyone following along, we will use this zoom link today: https://influxdata.zoom.us/j/92067511100. 

The program starts in 2 hours from now (1PM local time) and I expect presentations to go for about 2 hours -- we'll then have discussion groups which will likely be hard to zoom, and then we'll conclude around 5PM local time

**Comment by alamb** [comment]
Update: We were not able to record the entire session. 

Here are the slides from the prepared talks:
* [2024-03-25_ DataFusion Meetup - Introduction, Agenda, Remarks.pdf](https://github.com/apache/arrow-datafusion/files/14751567/2024-03-25_.DataFusion.Meetup.-.Introduction.Agenda.Remarks.pdf) - @alamb  ([video recording](https://www.youtube.com/watch?v=q1N3pH3tFw8))
* [Andy Grove @ DataFusion Meetup.pdf](https://github.com/apache/arrow-datafusion/files/14751568/Andy.Grove.%40.DataFusion.Meetup.pdf) - @andygrove 
* [Arroyo And DataFusion.pdf](https://github.com/apache/arrow-datafusion/files/14751569/Arroyo.And.DataFusion.pdf) @jacksonrnewhouse
* [datafusion-meetup-devind.pptx.pdf](https://github.com/apache/arrow-datafusion/files/14751570/datafusion-meetup-devind.pptx.pdf) - @devinjdangelo 
* [Replacing Spark with DataFusion.pptx.pdf](https://github.com/apache/arrow-datafusion/files/14751571/Replacing.Spark.with.DataFusion.pptx.pdf) - @Omega359 
* [SDF_Datafusion_DC2024Meetup.pdf](https://github.com/apache/arrow-datafusion/files/14751572/SDF_Datafusion_DC2024Meetup.pdf) @sadboy  and Lukas Schulte
*  [Lean-stack_data_engineering_with_datafusion_and_more.pdf](https://github.com/apache/arrow-datafusion/files/14767106/Lean-stack_data_engineering_with_datafusion_and_more.pdf) - @houqp

**Comment by alamb** [comment]
A common theme that came up several times was the desire for better support for nested / json / jsonb like structures -- aka https://github.com/apache/arrow-datafusion/issues/7845

**Comment by alamb** [comment]
Another thing that people mentiones several times was organizing other meetups in different locations. It sounds ilke maybe @gruuya  had spoken about potentially a meetup in Europe somewhere. It would probably be cool to do meetups also on the east/west coast of the US as well as in China (if not multiple locations).

Now we just need some one to volunteer to orgranize such events 🙏 I would be happy to help generate awareness / marketing ...

**Comment by alamb** [comment]
ANd with that I think I am going to claim this was a success; Thanks to everyone who came and participated!

**Comment by alamb** [comment]
I updated the list of talks in https://github.com/apache/arrow-datafusion/discussions/8522#discussioncomment-8909699  include [Lean-stack_data_engineering_with_datafusion_and_more.pdf](https://github.com/apache/arrow-datafusion/files/14767106/Lean-stack_data_engineering_with_datafusion_and_more.pdf) from  @houqp
